carbon-design-system/carbon
# feat(react): update icons to next version

This PR updates our codemod, migration docs, and our codebase to the new icon API.

#### Changelog

**New**

**Changed**

- Update React codebase with new icons
- Update our codemod to handle some situations that came up when running against our codebase
- Update our migration docs with more manual steps / instructions

**Removed**

#### Testing / Reviewing

- Verify that icons in Storybook render/look as expected (typically the 16x16 variant of the icon)
- Verify stories in Storybook that use icons work as expected (like in Button)
